### PR TITLE
[Manual] [Backport 2.x] New management overview page and rename stack management to dashboard…

### PR DESCRIPTION
**Pull request by Hailong-am**
Backport https://github.com/opensearch-project/OpenSearch-Dashboards/commit/d8f0c4834e90d181c30c6f178d7957bb2b64b295 from https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4287.

### CODE REVIEW

**Comment by Hailong-am** [comment]
@ashwin-pc @manasvinibs @ananzh Auto backport failed, i have create this manual backport, please help to review, thanks!

**Review by manasvinibs — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Add category option within groups for context menus ([#4144](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4144))
 - [Saved Object Service] Add Repository Factory Provider ([#4149](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4149))
 - Enable plugins to augment visualizations with additional data and context ([#4361](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4361))
+- New management overview page and rename stack management to dashboard management ([#4287](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4287))
```
Comment (ZilongX) [inline CHANGELOG.md:19]: Do we need to update the PR number to be the backport one (4528) or keep using the original one is good enough ?

Comment (manasvinibs) [reply CHANGELOG.md:19]: Autocut PR's anyway skip the change log entry and even though manual backport ones has the opportunity to update the PR number in the change log, I see most of the existing ones in 2.x pointing to the original PR.

Comment (AMoo-Miki) [reply CHANGELOG.md:19]: No. Original PR stands as it has the most accurate history.

**Review by AMoo-Miki — APPROVED** [review]

**Pull request merged**